server/webapp/WEB-INF/rails.new/webpack/views/artifact_stores/artifact_stores_state.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const Stream = require('mithril/stream');
+
+class ArtifactStoresState {
+
+  constructor() {
+    this._artifactStores = undefined;
+    this._pluginInfos    = undefined;
+    this._error          = false;
+    this._loading        = true;
```

Comment:
this._variable will actually expose the variable onto the object. So even if we have getter/setter methods to operate on the variable, the variable is exposed.
Can we say?

```
class ArtifactStoresState {
  let _error = false;
  constructor() {}
  get error() {
      return _error; 
  }
}
```